eng/pipelines/bicep-warnings.yml

```diff
@@ -0,0 +1,46 @@
+trigger: none
+
+pr:
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - '**/*.bicep'
+      - eng/pipelines/bicep-warnings.yml
+
+stages:
+  - stage: LintBicepFiles
+    displayName: Lint Bicep Files
+
+    pool:
+      vmImage: azsdk-pool-mms-ubuntu-2204-1espt
```

Comment:
For this I was suggesting something like:

```
  variables:
    - template: /eng/pipelines/templates/variables/image.yml

...

  pool:
    name: $(LINUXPOOL)
    vmImage: $(LINUXVMIMAGE) 
```

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-js/pull/30904/commits/c62f166432ab5a59465eb85726c5661502e854c5